queue_services/entity-filer/tests/unit/worker/test_amalgamation_application.py

```diff
@@ -127,31 +127,30 @@ async def test_short_form_amalgamation_application_process(app, session, amalgam
 
     create_share_class(amalgamating_business_1)
 
-    # TODO: implement create_party() & create_entity_role()
```

Comment:
These actually already exist, they've just been renamed to `create_entity_person` and `create_entity_role`. Otherwise the implementation is the same (as far as I can tell).